src/pe_reports/data/db_query.py

```diff
@@ -20,10 +20,10 @@
 
 def show_psycopg2_exception(err):
     """Handle errors for PostgreSQL issues."""
-    err_type, traceback = sys.exc_info()
-    line_n = traceback.tb_lineno
-    logging.error(f"\npsycopg2 ERROR: {err} on line number: {line_n}")
-    logging.error(f"psycopg2 traceback: {traceback} -- type: {err_type}")
+    err_type, err_obj, traceback = sys.exc_info()
+    logging.error(
+        f"Database connection error: {err} on line number: {traceback.tb_lineno}"
+    )
```

Comment:
One thing that I overlooked during my initial review is that we no longer use f-strings in our logging messages- [this post](https://www.siegescape.com/blog/2019/10/3/be-careful-with-your-f-strings) is a good short read on why we do that.

Please go through all of your logging statements and ensure that there are no f-strings in them.  If there are, you should either fix them now (preferable) or create an issue to fix them in the future.

```suggestion
    logging.error(
        "Database connection error: %s on line number: %s", err, traceback.tb_lineno
    )
```

Reply:
@dav3r addressed f-strings in the following commit: [5ad2c0b](https://github.com/cisagov/pe-reports/pull/114/commits/5ad2c0bf0ab8cf21b2d9adfb69c9158ac9f0cebd)

Reply:
Those look great now, aside from one minor thing that I'm about to note as a separate item.